scripts/update-app-download-links.js

```diff
@@ -15,6 +14,7 @@ module.exports = function updateAppDownloadLinks () {
   if (!matches.length) {
     // No downloads found for the detected OS, probably android/ios
     // Hide all links and let them click view all files if wanted
+    links[0].parentNode.style.display = 'none'
```

Comment:
Looks like this is a leftover change from a different PR.

Reply:
yes sorry about that

Reply:
There is another commit that has the fix for this issue though.